src/crudeditor-lib/i18n/de.js

```diff
@@ -47,7 +47,8 @@ const common = {
   "crudEditor.found.items.message": "{count} Datens\u00E4tze gefunden",
   "crudEditor.range.from": "von",
   "crudEditor.range.to": "bis",
-  "crudEditor.confirm.action": "Bestätigen"
+  "crudEditor.confirm.action": "Bestätigen",
+  "crudEditor.pagination.goToPage": "Gehe zu Seite"
```

Comment:
Translation does not look consistent with English one (it is too long for the button).